frontend/src/main/components/Leaderboard/LeaderboardTable.js

```diff
@@ -1,8 +1,7 @@
 import OurTable from "main/components/OurTable";
-import { hasRole } from "main/utils/currentUser";
 
 // should take in a players list from a commons
-export default function LeaderboardTable({ leaderboardUsers , currentUser }) {
+export default function LeaderboardTable({ leaderboardUsers }) {
```

Comment:
You removed `currentUser` as a prop, which is a good thing to do.

But that suggest that it should also be removed in all of the places where `LeaderboardTable` is used.  See below.